packages/orbit-components/.storybook/preview.ts

```diff
@@ -29,6 +29,25 @@ const preview: Preview = {
   decorators: [orbitDecorator],
   parameters: {
     viewport: { viewports },
+    a11y: {
+      element: "#storybook-root",
+      config: {
+        rules: [
+          {
+            id: "aria-required-attr",
+            selector: "*:not(div.text-heading-title0[role='heading'])",
```

Comment:
Same here, we can add a custom class to the heading. For instance, this selector could target headings on the Heading story, and we want to avoid that.